test/clj_gatling/simulation_test.clj

```diff
@@ -556,8 +646,8 @@
         result (group-by :name
                          (run-two-scenarios main-scenario second-scenario :concurrency 10 :requests 100))
         count-requests (fn [name] (reduce + (map #(count (:requests %)) (get result name))))]
-    (is (approximately== (count-requests "Main") 66 :accuracy 20))
-    (is (approximately== (count-requests "Second") 33 :accuracy 20))
+    (is (approximately== (count-requests "Main") 66 :accuracy 25))
+    (is (approximately== (count-requests "Second") 33 :accuracy 25))
```

Comment:
This was because I was getting frequent but flaky test failures with only 20% difference allowed, which I assume @mhjort was already aware of.